resources/clients/react/elros/README.md

```diff
@@ -32,48 +32,48 @@ structure, make it yours.
 1. Run `npm run dev`.
 
 ## Build
-1. Run `npm run build`
+1. Run `npm run build`.
 
 ## Understand the features
```

Comment:
```suggestion
## Understand the features
The following brief descriptions of key features will help you better understand this application.
```

Reply:
Ehhh, I can take or leave this one. Follow up PR and run it by editor.